src/Command/CreateClassCacheCommand.php

```diff
@@ -26,6 +26,8 @@
 /**
  * NEXT_MAJOR: Remove this class.
  *
+ * @deprecated since version sonata-project/admin-bundle 3.x and will be removed in 4.0.
```

Comment:
3.39.0 on both sides, but keep the package name please :)

Reply:
What's the point to keep the package name? You always know where the file is placed and what the namespace the class has.